keadm/cmd/keadm/app/cmd/edge/join_others.go

```diff
@@ -60,9 +59,6 @@ func AddJoinOtherFlags(cmd *cobra.Command, joinOptions *common.JoinOptions) {
 		fmt.Printf("mark flag required failed with error: %v\n", err)
 	}
 
-	cmd.Flags().StringVarP(&joinOptions.RuntimeType, common.RuntimeType, "r", joinOptions.RuntimeType,
```

Comment:
Please remove the constant common.RuntimeType and relevant commands: `keadm reset`, `keadm config iamges`

Reply:
> Please remove the constant common.RuntimeType and relevant commands: `keadm reset`, `keadm config iamges`

Sorry, could you please point out the specific location which need to removed? It seems like I have already deleted the constant common.RuntimeType.

Reply:
Sorry, I didn't see it clearly. I thought you only deleted the flag of the edge join command.